desc/compat.py

```diff
@@ -331,135 +330,3 @@ def _get_new_coeffs(fun):
     eq_rotated.axis = eq_rotated.get_axis()
 
     return eq_rotated
-
-
-def contract_equilibrium(
```

Comment:
This function was already on master and should not be deleted

Reply:
It's not. There is another instance of contract_equilibrium that has the latest from the last commit.